CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ version directory, and  then changes are introduced.
 
 ## [v4.6.0] WIP
 
+### Changed
+
+- Update `giantswarm-critical` priority class manifest to use `v1` stable.
```

Comment:
I wonder if this is change or fix?

Reply:
Priority classes moved to stable in 1.14 but the v1alpha1 is a bug it should be v1beta1. I'll change to fixed.

Reply:
Done.

Reply:
Yeah but then isn't this 4.5.1? Sorry for being nitpicky. Maybe that doesn't matter for our internal library.

Reply:
No problem but I'm not sure yet how this will be released. I moved my change to the new WIP version that @MarcelMue added earlier.

Reply:
Sure this is WIP. Ignore me. Thanks.